README.md

```diff
@@ -35,4 +35,10 @@ Please see [PerformanceIssues-Windows.md](PerformanceIssues-Windows.md).
 ## Security Concerns
 
 If you need to report a bug that an attacker could take advantage of, please instead email
-security@elastic.co so we can responsibly address the issue.
+security@elastic.co so we can responsibly address the issue. Further reading [here](https://www.elastic.co/community/security)
+
+## Asking questions
+
+* You can ask a question in the [forum](https://discuss.elastic.co/c/security/endpoint-security).
+* You are welcome to join [Elastic Stack Community](https://elasticstack.slack.com) slack and ask for help on the `#endpoint-security` channel.
```

Comment:
Not a blocker, but I prefer Slack because I find we can resolve customer issues more quickly.  Maybe if we put it first, more folks will choose it.

```suggestion
* You are welcome to join [Elastic Stack Community](https://elasticstack.slack.com) slack and ask for help on the `#endpoint-security` channel.
* You can ask a question in the [forum](https://discuss.elastic.co/c/security/endpoint-security).
```